modules/reference/pages/console/programmable-push-filters.adoc

```diff
@@ -1,18 +1,17 @@
 = Programmable Push Filters
-:description: Learn how to filter Kafka records in Redpanda Console based on your provided JavaScript code.
+:description: Learn how to filter Kafka records based on your provided JavaScript code.
```

Comment:
This is specifically about Redpanda Console. We should undo these changes. If this page needs to be included in cloud docs, conditionalize the content.

Reply:
No Vlado confirmed that it's not specific to console and that this could be written more generically for both cloud & console.

Reply:
But the plan is to have both self-hosted and cloud docs sourced from the same content. We can use attributes to reference the product name so that it's more helpful for both users. `{ui}` could be replaced by Redpanda Console in SH docs and Redpanda Cloud in cloud docs.

Also, this is now incorrect. It's Redpanda Console's filter engine. You're only filtering messages on Console's side. For cloud docs this could say Redpanda Cloud or we could just remove 'Redpanda' and refer to the filter engine.

> You can use Redpanda filter engine

Reply:
@micheleRP and @JakeSCahill, I've added the {ui} attribute in all cases except for the final reference, which now says 'You can use the filter engine.'